install-keep-dashboard.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
+DONE_START='\n\e[1;32m' # new line + bold + green
+DONE_END='\n\n\e[0m'    # new line + reset
+
+WORKDIR=$PWD
+
+printf "${LOG_START}Starting Keep Dashboard deployment...${LOG_END}"
+
+printf "${LOG_START}Preparing keep-core artifacts...${LOG_END}"
+
+cd $WORKDIR/keep-core/solidity
+ln -sf build/contracts artifacts
+
+printf "${LOG_START}Preparing keep-ecdsa artifacts...${LOG_END}"
+
+cd $WORKDIR/keep-ecdsa/solidity
+ln -sf build/contracts artifacts
```

Comment:
In the keep token dashboard, we use tbtc contracts as well. I think we need symlink `artifacts` to `build/contracts`.

```
cd $WORKDIR/tbtc/solidity
ln -sf build/contracts artifacts
```

Reply:
Thanks, I just added it.